src/vs/base/browser/ui/iconLabel/iconLabel.ts

```diff
@@ -100,6 +101,8 @@ export class IconLabel extends Disposable {
 	private descriptionNode: FastLabelNode | HighlightedLabel | undefined;
 	private descriptionNodeFactory: () => FastLabelNode | HighlightedLabel;
 
+	private hoverDelegate?: IHoverDelegate;
```

Comment:
I somewhat prefer not to use `?` in my classes, so I suggest to do `IHoverDelegate | undefined = undefined`